app/models/breakeven_table_question.rb

```diff
@@ -0,0 +1,41 @@
+class BreakevenTableQuestion
+  def initialize(breakeven_data)
+    @breakeven_data = JSON.parse(breakeven_data) if breakeven_data.present?
+  end
+
+  def report
```

Comment:
Can you please put a comment on this method stating its purpose and include the super nice stub that you made for Liz as an example of the output. Shame to see that go away!